sdks/python/apache_beam/io/gcp/pubsub.py

```diff
@@ -33,8 +33,9 @@
 from __future__ import absolute_import
 
 import re
+from builtins import object
 
-from six import text_type
+from past.builtins import basestring
```

Comment:
It seems that on Python 2 this change translates to changing the `element_type` type from `unicode` to `bytes`/`str` in line 176. 
@chamikaramj Do you have an intuition if this could bite us?
CC: @altay

Reply:
Correction: `past.builtins.basestring` is configured to mimic a superclass of both `str` and `unicode` in Python 2. So possibly there is no significant change in behavior but I am not 100% sure.

Reply:
It's indeed a superclass of both `str` and `unicode`.  Only in https://github.com/apache/beam/pull/5373#discussion_r195623409 I observed the real need to use this superclass. However for this example, it seems like `past.builtins.unicode` would be a more compatible replacement (https://pythonhosted.org/six/#six.text_type) since I can't recall an explicit reason to use basestring instead of unicode here.

Reply:
https://github.com/apache/beam/pull/6144 cleans this up here as well as in the place you mentioned, since we have reverted `# cython: language_level=3`.